source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -47,6 +47,20 @@ Side note 2. [If you haven't joined the Ember Community on Discord, do so today]
 
 ---
 
+## [Reason #2. Stable and More Reliable Than Ever 💪](https://emberjs.com/ember-community-survey-2019/#MS_Q401)
+
+According to the results of the latest community survey, a large proportion of the community has kept their Ember apps up to date with recent releases. Most Ember apps beyond 3.x were currently locked in at [version 3.7, 3.4 and 3.8](https://emberjs.com/ember-community-survey-2019/#MS_Q401).
+
+Seeing that a great number of Ember apps has managed to keep up with some of the latest is quite striking. It shows that Ember continues to deliver on its promise of **Stability without Stagnation**. As a framework which has been thriving in the fast-paced front-end ecosystem for over seven years, it always promoted itself as a stable and reliable solution for developers who want to make sure that the applications they're building are not only functional, but also still easy to maintain and upgrade tomorrow.
+
+A major contributing factor for the project's stability is the fine-tuned release process and the straight forward upgrade path. The release process has been refined several times in the past. Since the old ages of Ember 1.0 more than 5 years ago, a six-week release cycle had been introduced, [each one of them including beta and canary builds, as well as feature flags for not yet released, public API changes](https://blog.emberjs.com/2013/09/06/new-ember-release-process.html). A few years later, the [introduction of LTS releases](https://blog.emberjs.com/2016/02/25/announcing-embers-first-lts.html) made it even easier for developers to keep up with important updates and bug fixes, while still being able to upgrade their applications smoothly in their own time.
```

Comment:
Can you change "straight forward" to "straightforward"?

I might try to reorder the words to make your explanations of release processes more clear. How about:

"A 6-week release cycle started with Ember 1.0 (that was more than 5 years ago!) and introduced us to concepts such as beta and canary builds, as well as feature flags for public API changes that aren't yet released. A few years later, LTS releases made it even easier for [...]."

Reply:
Although not a part of the survey, I think a couple of additional reasons that developers can confidently upgrade to latest versions at any time are (1) tests and (2) codemods. Do you think there's merit to explaning/briefly mentioning them in your section?

Reply:
Thank you for the suggestion! I'll definitely add a mention on the codemods - can you also elaborate on this point?

>  I think a couple of additional reasons that developers can confidently upgrade to latest versions at any time are (1) tests

Are you referring to the fact, that end-user applications have confidence in the upgrade based on the results of their own app test suite or are you referring to the tests that ember.js / ember-data, etc. themselves as a library have that try to ensure that the packages work as expected in e.g. different node environments?

Reply:
Hmm, good question. I was thinking of the former (their own app test suite) when I had made the suggestion. I don't have enough experience with the core packages to suggest which of the two would be better to go over.

Reply:
Super, I just added a mention of ember-try as well to highlight how users even have an easy way to test their applications against new releases ahead of time

Reply:
Cool. Thank you!